### PR TITLE
docs-infra: Remove `rev:true` property

### PR DESCRIPTION
**Pull request by sonukapoor**
This property was used when the data was stored in the `Firebase` database
and `rev:true` meant that an admin has reviewed and approved the data. It
is however not used anymore.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx, @sonukapoor! BTW, you also need to remove `rev` from the other places it is used. Eg.:
- [resource-list.component.html](https://github.com/angular/angular/blob/2d5bb26b028397990f0dcd677bf5c27f57be088a/aio/src/app/custom-elements/resource/resource-list.component.html#L16)
- [resource.model.ts](https://github.com/angular/angular/blob/39e066add18c48af24db80ee00b775d73d926dab/aio/src/app/custom-elements/resource/resource.model.ts#L20)
- [resource.service.spec.ts](https://github.com/angular/angular/blob/d349cd91b162336515f413c81e0dc9afcf1cc067/aio/src/app/custom-elements/resource/resource.service.spec.ts)

##
For context:
The `rev` property has been in the initial commit that introduced `resource.json` (196203f6d76ac5240d1485dbd9701bfb026eab56) and has been added in all new entries since (always with value `true`). IIRC, this field is a remnant from back when this data was stored in a Firebase Database and the `rev` field indicated whether the entry has been reviewed/approved by a DevRel lead or something. Now that the data is kept in the repository and the reviewing is done as part of the corresponding PR, this field is no longer necessary.

@sonukapoor, can you please add that info in the commit message ☝️

**Comment by gkalpak** [comment]
BTW, I believe the commit should be of type `docs-infra`, not `docs`.

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -713,132 +612,112 @@
...
-            "rev": true,
             "title": "Angular.de (German)",
             "url": "https://angular.de/"
           }
         }
       }
     }
   }
-}
+}
```
[the remark this answers is not in the document]
Comment (sonukapoor) [reply aio/content/marketing/resources.json]: Fixed in https://github.com/angular/angular/pull/37181/commits/2491746815f3e31ae0425acb28628bfb8aa3f472

File: aio/content/marketing/resources.json [hunk 2]
```diff
@@ -7,38 +7,32 @@
...
             "url": "https://www.madewithangular.com/"
           },
           "angular-subreddit": {
             "desc": "An Angular-dedicated subreddit.",
-            "rev": true,
             "title": "Angular Subreddit",
             "url": "https://www.reddit.com/r/Angular2/"
           },
-          "angular-devto" : {
+          "angular-devto": {
```
[the remark this answers is not in the document]
Comment (sonukapoor) [reply aio/content/marketing/resources.json:28]: Thanks @gkalpak - I have made the changes to the component / model and spec file. 

https://github.com/angular/angular/pull/37181/commits/2491746815f3e31ae0425acb28628bfb8aa3f472

**Review by IgorMinar — APPROVED** [review]
LGTM! Thank you!

Reviewed-for: global-approvers

**Review by gkalpak — APPROVED** [review]
Thx, @sonukapoor ✨ 

FYI, I've massaged the commit message a bit and rebased on master.

**Comment by sonukapoor** [comment]
Thank you @gkalpak

**Pull request closed without merging**